react/tic-tac-toe/src/redux/auth/actions.js

```diff
@@ -24,6 +25,7 @@ const actionsCreators = {
         Authorization: session.token
       });
     } else {
+      console.log('tioo!!!');
```

Comment:
Not your uncle, and even if he was reviewing these files, I don't think he would allow a `console.log()`